### PR TITLE
Add roadmap section to the README

### PR DESCRIPTION
**Pull request by surajssd**
This is more of list of tasks we wanna accomplish, the tasks are in decreasing order of their priority.

### CODE REVIEW

**Review by concaf — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 The goal of OpenCompose is to make it easier for developers to on-board to Kubernetes.
 OpenCompose is a declarative higher level abstraction for specific Kubernetes resources.
 
-Very simple idea, isn't it? 
+Very simple idea, isn't it?
```
Comment (concaf) [inline README.md:8]: nit: let's put this in another PR

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Please send any PRs, issues or RFCs to improve this project.
 You can retrieve binaries for Linux, macOS and Windows on our [GitHub release page](https://github.com/redhat-developer/opencompose/releases).
 
 ```sh
-# Linux 
+# Linux
```
Comment (concaf) [inline README.md:24]: and this too

**Comment by concaf** [comment]
@surajssd how about putting this in a separate `ROADMAP.md`?

**Comment by kadel** [comment]
closing in favor of https://github.com/redhat-developer/opencompose/issues/131
